scintilla/win32/scintilla.mak

```diff
@@ -19,7 +19,7 @@ LEXLIB=Lexers.lib
 
 LD=link
 
-CRTFLAGS=-D_CRT_SECURE_CPP_OVERLOAD_STANDARD_NAMES=1 -D_CRT_SECURE_NO_DEPRECATE=1
+CRTFLAGS=-D_CRT_SECURE_CPP_OVERLOAD_STANDARD_NAMES=1 -D_CRT_SECURE_NO_DEPRECATE=1 -DSCI_NAMESPACE=1

```

Comment:
This should be another commit. First commit for adding the missing namespaces, then another for enabling those namespaces.